header the met landing

VerJane23 opened the pull request:
a Pull Request with a [DEMO LINK](https://VerJane23.github.io/layout_landing-page/)

anastasiiavorobiova reviewed (changes requested):
Great start! Try to fix validation errors and go ahead! 
<img width="1293" alt="Screenshot 2023-11-16 at 14 58 55" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/68bd333d-bca9-468d-b285-b87e5a53ce46">

VerJane23 reviewed (commented):
виправлені обидві помилки, дякую

anastasiiavorobiova reviewed (changes requested):
Awesome work! The task will be approved after implementing the whole landing sections

VerJane23 commented:
Thank you!

VerJane23 commented:
The Metropolitan Museum landing page project was made using the BEM website development methodology and using SASS preprocessors. Smooth transitions, as well as transformation and animation elements are used for built-in effects. Grid placement is mainly used to position project elements.
The project page is located at: https://verjane23.github.io/layout_landing-page/

l4st1m0za reviewed (changes requested):
Good job! But here are some things that you need to improve:

1) When I hover over the phone I have other style of the pointer and the phone becomes invisible
<img width="244" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/69cf9d0e-49d0-4d4c-997b-320e53915f00">

2) Also check indents between the phone numbers, as in figma

3) Your placement of the menu button and exit menue button is wrong. This buttons should always be located on the same place

4) Photos are in the bad quality. Use .svg for all of them
<img width="368" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/159503e0-a21b-45c8-94d8-f20811cb46db">

5) When I click on the photos from this block I get teleported to the start of the page
<img width="678" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/e4debd8f-c5c9-4270-baf0-347c01d4dc6d">

6) Whenn I click on the text input I get padding for the placeholder which disappears after the click
<img width="376" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/498240a6-08c9-43d3-a172-b510e12e7d97">

7) Your footer is scrollable

8) Disable scroll in the menue
<img width="1255" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/6a00241c-568c-4494-8629-5ca606981714">

9) This elements should have hover effect 
<img width="506" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/bbb29eef-fa2c-45cc-a5fb-78e912471f04">

10) Make paddings similar in different screen mods
<img width="590" alt="Screenshot 2023-12-05 at 22 01 33" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/d1fdd431-d353-41e6-8b7d-8966eb64ad7b">

11) Theck the layout here in tablet mod
<img width="580" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/67686312/dadede87-d4e3-4dde-8c89-a051bf3622cf">

12) Also make so that all of your animations, on buttons, menue, images, etc. should have the same speed. Now half of your animations don't have speed at all, they just change in the matter of second

VerJane23 commented:
Thank you for your feedback. Each item has been checked and corrected. 
1. When you hover, the “phone” pointer disappears, an “arrow” and a hint text appear. When you hover over the tooltip, the contact text is highlighted and “phone” appears again as an invitation to make a call. 
2. Added 8px separator. 
3. In this case, the author of the design dictates uniform “menu” sizes, namely the “mobile version” for the project. Thus, the header paddings and menus of the tablet and desktop versions do not have the same sizes. 
4. Photos of the exhibition have been replaced. 
5. Now, when you click on a photo, the cursor moves to a button asking you to go to view the entire exposure. 
6. A smooth change effect has been added to form fields. 
7. Footer is fixed. 
8. Menu scrolling has been removed. 
9. Added hover effect to contacts. 
10-11. Absolute data is excluded, content is aligned. 
12. All site animations are given the same value.

VerJane23 commented:
Thank you for your feedback. Each item has been checked and corrected. 
1. When hovering, the “phone” pointer was returned. 
3. Added a left indent to the form fields of 12 (16) px. 
4. 3. Fade effect has been added to the contacts section.

Esceype reviewed (changes requested):
Close to perfect!
Check all your images and links, all the items that have hover effects. It is should be smooth when the mouse enters and when it leaves, but at the moment there are some items that have not any transition when the mouse leaves.

VerJane23 commented:
Thank you for the lack of specifics. I really like looking for a needle in a haystack.

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,352 @@
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo-link">
+              <img src="./images/logoMet.svg"
+                   alt="met logo"
+                   class="top-bar__logo"
+              >
```
Need to fix code style everywhere 
```suggestion
              <img
                src="./images/logoMet.svg"
                alt="THE MET logo"
                class="top-bar__logo"
              >
```

volodymyr-soltys97 commented on src/index.html:
```diff
@@ -1,13 +1,352 @@
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo-link">
+            <img src="./images/logoMet.svg"
+                 alt="met logo"
+                 class="top-bar__logo"
+            >
```
The same as above, need to check and fix it everywhere

volodymyr-soltys97 reviewed (changes requested):
Almost done!
Let's improve your code style

denys-danyliuk reviewed (changes requested):
4'th point in checklist
<img width="528" alt="Знімок екрана 2023-12-08 о 14 27 47" src="https://github.com/mate-academy/layout_landing-page/assets/106310796/50e80c57-c15d-44e9-9919-0ae99c295cd3">

check the design, this text should have greater font-weight
<img width="252" alt="Знімок екрана 2023-12-08 о 14 28 30" src="https://github.com/mate-academy/layout_landing-page/assets/106310796/8909f190-e711-4b8f-89e4-65d1b2d480a0">

this images should also be links and have cursor pointer, 7'th point in checklist
<img width="1099" alt="Знімок екрана 2023-12-08 о 14 29 21" src="https://github.com/mate-academy/layout_landing-page/assets/106310796/f36beeef-392f-47e3-bf4f-c24265b0c1e5">

add paddings to last link
<img width="421" alt="Знімок екрана 2023-12-08 о 14 31 16" src="https://github.com/mate-academy/layout_landing-page/assets/106310796/b1271228-a4aa-443c-baaf-1ced7ec3ea6a">

check if you've added right color, because now it doesn't look like on a design
![Знімок екрана 2023-12-08 о 14 32 17](https://github.com/mate-academy/layout_landing-page/assets/106310796/65a76f36-964e-4181-81b9-0650e3ae6428)
<img width="460" alt="Знімок екрана 2023-12-08 о 14 32 21" src="https://github.com/mate-academy/layout_landing-page/assets/106310796/7603acdc-ad38-497d-81fb-bdfc479851ea">

denys-danyliuk reviewed (changes requested):
you should fix all the comment before requesting a new review

choose some other color, it's hard to tell if it's hovered or not
![Знімок екрана 2023-12-08 о 16 24 34](https://github.com/mate-academy/layout_landing-page/assets/106310796/6bafcdd1-f389-430f-bd08-271c797df81b)

these are not fixed
![Знімок екрана 2023-12-08 о 16 26 03](https://github.com/mate-academy/layout_landing-page/assets/106310796/5565d3a2-afe0-497a-9f50-f9bd3ff8e89a)
![Знімок екрана 2023-12-08 о 16 25 23](https://github.com/mate-academy/layout_landing-page/assets/106310796/cf3957b4-9343-41b6-a8a4-817305c062ea)
![Знімок екрана 2023-12-08 о 16 25 34](https://github.com/mate-academy/layout_landing-page/assets/106310796/144513d4-089d-4e85-aade-359cca1fd662)
![Знімок екрана 2023-12-08 о 16 28 29](https://github.com/mate-academy/layout_landing-page/assets/106310796/f6f20c23-824f-44f5-a57b-145cbe8ff481)

TarasHoliuk reviewed (approved):
:fire: Approved :heavy_check_mark:

VerJane23 commented:
Thank you
